.travis.yml

```diff
@@ -72,7 +72,8 @@ script:
   - export CGO_ENABLED=$TRAVIS_CGO_ENABLED
   - DCO_VERBOSITY=-q ../project/script/validate/dco
   - ../project/script/validate/fileheader ../project/
-  - travis_wait ../project/script/validate/vendor
+  - travis_wait 20 bash -c '../project/script/validate/vendor > /tmp/vendor-validate.log 2>&1 || (cat /tmp/vendor-validate.log; exit 1)';
```

Comment:
looks like the problem may be the `bash -c` and `travis_wait` not checking subprocesses https://github.com/travis-ci/travis-ci/issues/8526#issuecomment-402772421

Reply:
`travis_wait bash -c 'exit 1'` is ok. anyway, travis_wait is good but the validate doesn't work well.